src/css/font.less

```diff
@@ -9,18 +9,17 @@
 
 @basic:~ "";
 .font-srcs() when not (@basic) {
-  src: url(/fonts/Symbola.eot);
+  src: url(../fonts/Symbola.eot);
```

Comment:
These are now the actual, relative paths to the font files. `lessc` can now resolve them properly when building.